Functions/Args and kwargs/args_kwargs.py

```diff
@@ -21,16 +21,18 @@ def random_dialogue(place, *args, **kwargs):
 
 
 def cat(food, *args, state='still hungry', action='meow', breed='Siamese'):
-    print("-- This cat would", action, end=' ')
-    print("if you gave it", food)
-    print("-- Lovely fur, the", breed)
-    print("-- It's", state, "!")
+    print(f"-- This cat would {action}", end=' ')
+    print(f"if you gave it {food}")
+    print(f"-- Lovely fur, the {breed}")
+    print(f"-- It's {state}!")
     for arg in args:
         print(arg.upper())
 
+
 # Add a list of phrases that will be capitalized.
 phrases = ['It is too fat.', 'You are feeding your cat too much.']
 # Add a dict of keyword arguments.
 keywords = {'state': 'fat', 'action': 'eat', 'breed': 'Maine Coon'}
+keywords.values()
```

Comment:
what does this do?

Reply:
Actually nothing, that is a leftover from some crazier design